plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -1950,41 +1943,20 @@ describe('PersonState.update()', () => {
                 uuid.toString(),
                 ['first']
             )
-            const second: Person = await hub.db.createPerson(
-                timestamp,
-                {},
-                {},
-                {},
-                teamId,
-                null,
-                false,
-                uuid2.toString(),
-                ['second']
-            )
+            await hub.db.createPerson(timestamp, {}, {}, {}, teamId, null, false, uuid2.toString(), ['second'])
 
             const state: PersonState = personState({}, first)
             // break postgres
             const error = new DependencyUnavailableError('testing', 'Postgres', new Error('test'))
-            jest.spyOn(hub.db, 'postgresTransaction').mockImplementation(() => {
+            jest.spyOn(state, 'mergePeople').mockImplementation(() => {
```

Comment:
I gather this is because we didn't actually want to mock the `postresTransaction` method but rather the call to `client.query('COMMIT')` in it's body?

Reply:
oh could have actually kept it as postgresTransaction too, but we can re-review the decisions while working on person-state together